From 3d5f51f75823620b673b939e6bc5febd66ba83e0 Mon Sep 17 00:00:00 2001
From: Kevin Daudt <me@ikke.info>
Date: Wed, 20 Nov 2019 22:03:57 +0000
Subject: [PATCH] gallium: fix broken if-macro for PIPE_ARCH_LITTLE_ENDIAN

[ae071434e][0] is a commit that was backported from master in 19.2.
That commit also copied logic from [f9f60da81][1], which causes a
compilation failure on ppc64le because the macro itself was not changed
as well.

Change the `#if` to an `#ifdef` to fix this issue.

Fixes #2131

[0]:https://gitlab.freedesktop.org/mesa/mesa/commit/ae071434e918586ab88a6533cd4a3b3f3dbd5a1f
[1]:https://gitlab.freedesktop.org/mesa/mesa/commit/f9f60da813e69aacf541d25a24622c896f15ba98
---
 src/gallium/auxiliary/gallivm/lp_bld_misc.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gallium/auxiliary/gallivm/lp_bld_misc.cpp b/src/gallium/auxiliary/gallivm/lp_bld_misc.cpp
index 1a47bab7e47..e05c9064990 100644
--- src/gallium/auxiliary/gallivm/lp_bld_misc.cpp
+++ src/gallium/auxiliary/gallivm/lp_bld_misc.cpp
@@ -705,7 +705,7 @@ lp_build_create_jit_compiler_for_module(LLVMExecutionEngineRef *OutJIT,
     */
    builder.setCodeModel(CodeModel::Large);
 
-#if PIPE_ARCH_LITTLE_ENDIAN
+#ifdef PIPE_ARCH_LITTLE_ENDIAN
    /*
     * Versions of LLVM prior to 4.0 lacked a table entry for "POWER8NVL",
     * resulting in (big-endian) "generic" being returned on
-- 
2.22.0

